lib/routes/sign.js

```diff
@@ -96,15 +97,25 @@ module.exports = function (log, isA, error, signer, db, domain) {
             duration: duration,
             generation: sessionToken.verifierSetAt,
             lastAuthAt: sessionToken.lastAuthAt(),
-            verifiedEmail: sessionToken.email
+            verifiedEmail: sessionToken.email,
+            deviceId: sessionToken.deviceId ?
+              sessionToken.deviceId.toString('hex') : null
           }
-        ).then(function(certResult) {
-          log.activityEvent('account.signed', request, {
-            uid: uid,
-            account_created_at: sessionToken.accountCreatedAt
-          })
-          reply(certResult)
-        }, reply)
+        )
+        .then(
+          function(certResult) {
+            log.activityEvent(
+              'account.signed',
+              request,
+              {
+                uid: uid,
+                account_created_at: sessionToken.accountCreatedAt
```

Comment:
While we're here, could you please log the device-id as part of this event as well?